tasks/db_deploy/orca_sql.py

```diff
@@ -413,9 +415,213 @@ def recovery_file_table_sql() -> TextClause:
+    """
+    )
+
+def files_table_sql() -> TextClause:
+    """
+    Full SQL for creating the catalog files table.
+
+    Returns:
+        (sqlalchemy.sql.element.TextClause): SQL for creating files table.
+    """
+    return text(
+        """
+        -- Create table
+        CREATE TABLE IF NOT EXISTS files
+        (
+          id                        text NOT NULL
+        , granule_id                text NOT NULL         
+        , name                      text NOT NULL
+        , orca_archive_location     text NOT NULL
+        , cumulus_archive_location  text NOT NULL
+        , key_path                  text NOT NULL
+        , ingest_time               timestamp with time zone NOT NULL
+        , etag                      text NOT NULL
+        , version                   text NOT NULL
+        , size_in_bytes             int8 NOT NULL
+        , hash                      text NULL
+        , hash_type                 text NULL
+        , CONSTRAINT PK_files PRIMARY KEY (id)
+        , CONSTRAINT FK_granule_file_fk FOREIGN KEY (granule_id) REFERENCES granules (id)
+        );
+
+        -- Comments
+        COMMENT ON TABLE files
+            IS 'Files that are in the ORCA holdings. (Latest version only)';
+        COMMENT ON COLUMN files.id
+            IS 'Internal ORCA file ID';
+        COMMENT ON COLUMN files.granule_id
+            IS 'Granule that the file belongs to refrences the internal ORCA granule ID.';
+         COMMENT ON COLUMN files.name
+            IS 'Name of the file including extension';
+         COMMENT ON COLUMN files.orca_archive_location
+            IS 'S3 Glacier bucket that the file object is stored in';
+         COMMENT ON COLUMN files.cumulus_archive_location
+            IS 'Cumulus S3 bucket where the file is thought to be stored.';
+         COMMENT ON COLUMN files.key_path
+            IS 'Full AWS key path including file name of the file (does not include bucket name) where the file resides in ORCA.';
+        COMMENT ON COLUMN files.ingest_time
+            IS 'Date and time the file was ingested into ORCA';              
+        COMMENT ON COLUMN files.etag
+            IS 'etag of the file object in the AWS S3 Glacier bucket.';
+        COMMENT ON COLUMN files.version
+            IS 'Version of the file in the S3 Glacier bucket';   
+        COMMENT ON COLUMN files.size_in_bytes
+            IS 'Size of the object in bytes';
+        COMMENT ON COLUMN files.hash
+            IS 'Hash of the object from Cumulus';
+        COMMENT ON COLUMN files.hash_type
+            IS 'Hash type used to hash the object. Supplied by Cumulus.';                 
+        -- Grants
+        GRANT SELECT, INSERT, UPDATE, DELETE ON files TO orca_app;
+    """
+    )
+
 
 # ----------------------------------------------------------------------------
-# ORCA SQL used for migration of schema v1 to schema v2
+# ORCA SQL used for migration of schema v2 to schema v3
```

Comment:
Again, really feels like we should still have the old migration code.

Reply:
Should we keep the codes for v1 as well? 

I removed all code for `v1 to v2` but can add that back.

Reply:
yes that is a good idea I think